py/confluent/build/download_artifacts/download_artifacts.py

```diff
@@ -0,0 +1,77 @@
+import json
+import re
+
+import requests
+
```

Comment:
remove space.

native imports

3rd party imports

confluent imports

(ordered alphabetically .. )

Reply:
Blank line is required between different types of imports? I will clean them alphabetically.

Reply:
thats the practice we are following. refer to other similar imports in release-tools.